doc/man3/EVP_PKEY_KEM_decapsulate.pod

```diff
@@ -0,0 +1,93 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_KEM_decapsulate_init, EVP_PKEY_KEM_decapsulate
+- Key decapsulation using a private key algorithm
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_KEM_decapsulate_init(EVP_PKEY_CTX *ctx);
+ int EVP_PKEY_KEM_decapsulate(EVP_PKEY_CTX *ctx,
+                              unsigned char *secret, size_t *secretlen,
+                              const unsigned char *wrapped, size_t wrappedlen);
```

Comment:
Why `EVP_PKEY_KEM`?  It's really not necessary to pile on more "prefixes", and I believe the rest of the function name is unique enough to not clash with anything.

Reply:
@levitte You are right, probably `EVP_PKEY_encapsulate{,_init}` and the `decapsulate` counterpart are better!

Apologies for my original misguided proposal!